### ISSUE TITLE
Cannot convert RandomForestClassifier when model labels is a 2d array

### ISSUE THREAD

**Issue by miguel-angel-monjas** [onnx/sklearn-onnx#656]
Hi there,

I've got a trained RandomForestClassifier. X_train.shape is (73578, 18). y_train.shape is (73578, 2). Initially, the model was trained using pandas dataframes, but I eventually converted them to numpy arrays. Both features and targets are integers. For further clarification, types of observations and targets are integers (dtype('int64'))

I follow the instructions:

```python
from sklearn.ensemble import RandomForestClassifier

model = RandomForestClassifier(n_estimators=30,
                               min_samples_leaf=2,
                               min_samples_split=2,
                               max_depth=15,
                               oob_score=True,
                               random_state=44)
model.fit(X_train, y_train)

from skl2onnx import convert_sklearn
from skl2onnx.common.data_types import Int64TensorType

initial_type = [('input', Int64TensorType([None, 18]))]
onx = convert_sklearn(model, initial_types=initial_type)

with open("cool_model.onnx", "wb") as f:
    f.write(onx.SerializeToString())
```

The conversion and serialization work, apparently, fine. However, when I try to load it again an run it using the ONNX Runtime:

```python
import onnxruntime as rt
sess = rt.InferenceSession("cool_model.onnx")
```

It fails:

```
[ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Exception during initialization: /onnxruntime_src/onnxruntime/
core/providers/cpu/ml/zipmap.cc:42 onnxruntime::ml::ZipMapOp::ZipMapOp(const onnxruntime::OpKernelInfo&) 
classlabels_strings_.empty() ^ classlabels_int64s_.empty() was false. Must provide classlabels_strings or 
classlabels_int64s but not both.
```

Any idea of why it is failing?

Thanks

**Comment by miguel-angel-monjas** [comment]
BTW, the subject of the issue could be wrong. The fact that the outcome of the model is made of two targets seems to be the most relevant issue, but could be anything

**Comment by miguel-angel-monjas** [comment]
Environment:
```
onnx==1.9.0
onnxconverter-common==1.8.1
onnxoptimizer==0.2.6
onnxruntime==1.7.0
scikit-learn==0.24.2
skl2onnx==1.8.0
```

**Comment by miguel-angel-monjas** [comment]
It was not a problem of the model outcome but simply of ZipMap (see [Converters with options](http://onnx.ai/sklearn-onnx/parameterized.html#zipmap)). Fixed
